.github/workflows/kind-e2e.yaml

```diff
@@ -67,6 +67,8 @@ jobs:
 
     - name: Install mirror, kind, knative + sigstore
       uses: sigstore/scaffolding/actions/setup@main
+      with:
+        version: v0.3.0
```

Comment:
Also please add this:
k8s-version 1.22

so like this:
```
with:
  version: "v0.3.0"
  k8s-version: "v1.22.x"
```

Reply:
Because 23 is being goofy:
https://github.com/sigstore/scaffolding/issues/81

Reply:
Gotcha, updated.